editor/components/post-featured-image/check.js

```diff
@@ -2,9 +2,12 @@
  * Internal dependencies
  */
 import PostTypeSupportCheck from '../post-type-support-check';
+import ThemeSupportCheck from '../theme-support-check';
 
 function PostFeaturedImageCheck( props ) {
-	return <PostTypeSupportCheck { ...props } supportKeys="thumbnail" />;
+	return <ThemeSupportCheck supportKeys="post-thumbnails">
```

Comment:
Nit (Style): My preference (and predominantly throughout the codebase) is for JSX to be placed on its own lines if multi-lined with parentheses, e.g.:

```jsx
return (
	<ThemeSupportCheck supportKeys="post-thumbnails">
		<PostTypeSupportCheck { ...props } supportKeys="thumbnail" />
	</ThemeSupportCheck>
);
```

If it's a convention we want to adopt, we could enforce with:

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md

Reply:
> My preference (and predominantly throughout the codebase) is for JSX to be placed on its own lines with parentheses if multi-lined, e.g.:

Seems reasonable to me 19160260e